### DISCUSSION TITLE
Lock Screen

### DISCUSSION THREAD

**Discussion by MarJose123** [filamentphp/filament#3967] [Showcase]
I created a filament package to have an ability to user to lock their access without logged out.

You can find the package here: https://github.com/MarJose123/filament-lockscreen

the only missing part I guess, is to detect if the user has any activity to the system or not.

Please let me know what you think, any and all feedback is much appreciated!
![image](https://user-images.githubusercontent.com/18107626/189592216-72059d09-e52d-4582-9561-dba55f90fd54.png)
![image](https://user-images.githubusercontent.com/18107626/189592239-80096240-06d1-47b0-9140-081aeae761df.png)

**Comment by danharrin** [comment]
Nice! Have you submitted it to our website?

**Reply by MarJose123** [reply]
Yes I will do that now.

**Comment by shovann-notebrains** [comment]
After screen locked my frontend urls also getting blocked.
How to solve that ??

**Reply by shovann-notebrains** [reply]
![image](https://github.com/user-attachments/assets/8953c1e6-d981-493a-b960-ae8f02fc4437)
Getting issue into the frontend side.
![image](https://github.com/user-attachments/assets/2cb7bbac-74a3-443c-ab77-437be0509a07)
This is admin side

**Reply by danharrin** [reply]
Please ask in the repo https://github.com/MarJose123/filament-lockscreen, not here
